### DISCUSSION TITLE
Add Keepassxc to the Mac App Store

### DISCUSSION THREAD

**Discussion by matchcom-cahors** [keepassxreboot/keepassxc#9570] [Ideas]
**Reasons for Adding KeepassXC to the Mac App Store:**

1. **Enhanced Security:** By making KeepassXC available in the Mac App Store, users can have confidence in the security and authenticity of the application. 

2. **Convenience for Non-Technical Users:** The Mac App Store provides is user-friendly and intuitive for individuals who are not well-versed in computer usage. By offering KeepassXC through the App Store, less technically inclined users can easily find, download, and install the correct version of the application without the need for manual installation processes and validation.

3. **Automatic Updates:** Another significant advantage of distributing KeepassXC through the Mac App Store is the ability to deliver automatic updates. Users will benefit from receiving timely security patches, bug fixes, and new features without the hassle of manually checking for updates or visiting external websites for the latest version. This ensures that users are always up to date with the most secure and stable version of KeepassXC.

Keepass has successfully made its way into the Microsoft App Store, which is truly excellent news. This development has allowed me to onboard numerous friends and family members, as they can now benefit from receiving automatic updates.

**Comment by droidmonkey** [comment]
Microsoft app store does not deliver automatic updates. That only applies to "modern" packaged apps which are generally C# based. 

Mac store requires us to sandbox the application which would prevent us from performing a lot of our best features.

**Reply by matchcom-cahors** [reply]
Oh man, the fact that there are no automatic updates in the Microsoft store is killing me right now. I kind of expected that. Thanks for the explanation!

Ok, too bad. But it seems that the App store can't be compared with the Microsoft store either.

**Reply by adriannieto-at-fortris** [reply]
Even with the limitations and if some features are lost it probably worth is for the sake of the security isnt it?

**Reply by Volker-Weissmann** [reply]
> for the sake of the security

I might be telling you something that you already know here, but the sandboxing would protect the rest of the machine from KeypassXC, not the other way around.

**Reply by adriannieto-at-fortris** [reply]
@Volker-Weissmann Yep, my concern was more about securing distribution than actual environment/app security  😄
